src/automations/checkInactivity.js

```diff
@@ -102,25 +107,29 @@ async function scrapeInactiveIssues(client, references, issues) {
   const iterator = issues[Symbol.iterator]();
 
   for (let issue of iterator) {
+    const issueNumber = issue.number;
+    const repoName = issue.repository.name;
+    const repoOwner = issue.repository.owner.login;
+    const issueTag = `${repoName}/${issueNumber}`;
+
     const inactiveLabel = issue.labels.find(label => {
       return label.name === client.cfg.activity.inactive;
     });
     if (inactiveLabel) continue;
+    if (!references.get(issueTag).review) continue;
```

Comment:
You need to check if `references.has(issueTag)` is true first before trying to fetch the issueTag from the references map. Not all issues have a reference, so you mind receive some errors like "Cannot read property review of undefined", because `references.get(issueTag)` is undefined